core/promise/dto.go

```diff
@@ -27,10 +27,10 @@ type SignedPromise struct {
 
 // Promise represents payment promise between two parties
 type Promise struct {
-	SerialNumber int
-	IssuerID     string
-	BenefiterID  string
-	Amount       money.Money
+	SerialNumber int    `storm:"id"`
+	IssuerID     string `storm:"index"`
+	BenefiterID  string `storm:"index"`
+	Fee          money.Money
```

Comment:
I would stick with 'Amount'. Promise usually is not actual 'Fee', but just promised 'Amount'.